ibis/backends/trino/tests/test_client.py

```diff
@@ -164,7 +164,9 @@ def test_table_access_database_schema(con):
     t = con.table("region", database=("tpch", "sf1"))
     assert t.count().execute()
 
-    with pytest.raises(exc.IbisError, match='Table not found: tpch."tpch.sf1".region'):
+    with pytest.raises(
+        exc.TableNotFound, match='Table not found: tpch."tpch.sf1".region'
```

Comment:
```suggestion
        exc.TableNotFound, match=r'Table not found: tpch\."tpch\.sf1"\.region'
```

Tiny nit, since `match` is a regex, let's make sure the dots are actual dots :)

(I realize this was pre-existing)